lib/logstash/runner.rb

```diff
@@ -63,15 +44,6 @@ def run(args)
         end
         return LogStash::Agent.run($0, agent_args)
       end,
-      "rspec" => lambda do
```

Comment:
ah good point. let's see if that is the best strategy to run that through runner.